index.html

```diff
@@ -7,10 +7,12 @@
 <p>The User Guides for the current versions are listed in the main menu, for all OMERO applications, including OMERO.insight, OMERO.web, OMERO.figure and OMERO in ImageJ and Fiji.</p>
 <p>The <a href="previous.html"> Guides for Previous Versions</a> page has links to archives of older versions.</p>
 <p>The <a href="resources.html">Training Course Material</a> page contains links to material for printing out and using in training sessions, ZIP archives of the PDFs for all the current versions, customised versions of some of the Guides and other resources such as  Omnigraffle&reg; and Word&reg; files.</p>
-<p>The current release versions of OMERO.insight are 5.1.0, 5.0.8 and 4.4.12. You can tell what version of OMERO.insight you are using by looking at the login screen.</p>
-<p>&nbsp;</p>
-<p><img src="images/homeVersion.jpg" alt="OMERO.insight Version" style="width: 100%;"/></p>
-<p>&nbsp;</p>
+<p>The current release versions of OMERO.insight are 5.1.0, 5.0.8 and 4.4.12. You can tell what version of OMERO.insight you are using by looking at the login screen, or selecting <span class="bold">About OMERO.insight</span> from the Help menu in OMERO.insight.</p>
+
+<p><img src="images/indexVersion.jpg" alt="OMERO.insight Version" style="width: 80%;"/></p>
+
+<p>All the Version 5 material on the User Help website applies to both version 5.0.8 and version 5.1, with the exception of the new features mentioned below. Clients for version 5.0.8 can be from the <a href="http://downloads.openmicroscopy.org/omero/5.0.8/">Version 5.0.8 Downloads</a> page. The User Help for the Version 5.0.8 release can be downloaded as <a href="http://downloads.openmicroscopy.org/help/archives/OMERO_User_Guides_5_0_8.zip">Version 5.0.8 PDF archive</a> (181 MB).</p>
```

Comment:
clients are available from - you have a word missing and 'downloaded from' isn't really necessary when you have 'downloads' in the link text